src/rank_sel/rank10sel.rs

```diff
@@ -106,14 +106,14 @@ impl<
         let inv_pos = rel_inv_pos + upper_block_idx * Self::UPPER_BLOCK_SIZE;
 
         let next_rel_inv_pos = *inv_ref.get_unchecked(rank / Self::ONES_PER_INVENTORY + 1) as usize;
-        let next_inv_pos = if next_rel_inv_pos > rel_inv_pos {
-            next_rel_inv_pos + upper_block_idx * Self::UPPER_BLOCK_SIZE
-        } else if next_rel_inv_pos < rel_inv_pos {
-            (upper_block_idx + 1) * Self::UPPER_BLOCK_SIZE
-        } else {
+        let next_inv_pos = match next_rel_inv_pos.cmp(&rel_inv_pos) {
+            std::cmp::Ordering::Greater => {
+                next_rel_inv_pos + upper_block_idx * Self::UPPER_BLOCK_SIZE
+            }
+            std::cmp::Ordering::Less => (upper_block_idx + 1) * Self::UPPER_BLOCK_SIZE,
```

Comment:
This change was suggested by clippy, if there is no performance relevant reason why the Greater case should be checked before the other ones, I think it improves readability